### ISSUE TITLE
sendReport method problem

### ISSUE THREAD

**Issue by 954514762** [WICG/webhid#116]
SendReport parameters have to be two?
Like this  selectedDevice.sendReport(reportId, reportData);

**Comment by nondebug** [comment]
Yes, there are two parameters to `sendReport`. The first parameter is the report ID, pass zero if the device does not use numbered reports. `reportData` contains the report data without the report ID byte.

What's the problem?

**Comment by 954514762** [comment]
Can carry multiple arguments

**Comment by 954514762** [comment]
Want to carry multiple parameters can achieve?

**Comment by chengweih001** [comment]
Could you please elaborate an example how you wish `sendReport` to look like?

**Comment by 954514762** [comment]
For example: sendReport (args1 args2, data, length, end), custom and so on parameters, without limiting the number

**Comment by nondebug** [comment]
I don't understand how adding parameters would be useful.

What are `args1` and `args2` used for? If you're trying to send data to the device, it needs to be inside the `reportData` parameter which needs to follow the report layout defined in the device's report descriptor.

What are `data`, `length`, and `end` used for? There's already a `reportData` parameter which is a DataView so it already includes the extent.

Can you tell us more about the problem you're having that caused you to file this feature request?
